### PR TITLE
improved(pair_sites)

### PR DESCRIPTION
**Pull request by agalitsyna**
min-max interjacent fragments added
pair-sites transferred to ops

### CODE REVIEW

File: bioframe/ops.py [hunk 1]
```diff
@@ -1324,3 +1324,102 @@ def count_overlaps(
     )
 
     return df_counts
+
+def pair_by_distance(
+        df,
+        min_sep,
+        max_sep,
+        min_interjacent=None,
```
Comment (nvictus) [inline bioframe/ops.py:1332]:
```suggestion
        min_intervening=None,
```
```suggestion
        min_interjacent=None,
```

File: bioframe/ops.py [hunk 2]
```diff
@@ -1324,3 +1324,102 @@ def count_overlaps(
...
+        max_interjacent=None,
```
Comment (nvictus) [inline bioframe/ops.py:1333]:
```suggestion
        max_intervening=None,
```
```suggestion
        max_interjacent=None,
```

**Comment by nvictus** [comment]
Just a suggestion to change the term "interjacent" to the more common term "intervening". Though both are [acceptable](https://www.dictionary.com/browse/interjacent?s=t)!

Otherwise looks ready to merge!

**Comment by agalitsyna** [comment]
Okay, no other interventions detected!

**Review by golobor — APPROVED** [review]

**Pull request merged**